src/api/content-rest-api/docs/CategoryPaging_list.md

```diff
@@ -0,0 +1,9 @@
+# CategoryPagingList
```

Comment:
Do you have correct name of that file? It looks like that file and ts file use two different approaches. Here you have _list but in ts you have List without _

Reply:
I think it is correct, I followed the pattern that other apis use e.g. tags api

Reply:
ah I see, not sure why we use two different approaches :D Then keep it as it is :)